joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstCreator.scala

```diff
@@ -254,7 +254,11 @@ class AstCreator(
   def astForMultipleRightHandSideContext(ctx: MultipleRightHandSideContext): Seq[Ast] = {
     if (ctx == null) return Seq(Ast())
 
-    val exprAsts = ctx.expressionOrCommands().expressionOrCommand().asScala.flatMap(astForExpressionOrCommand).toSeq
+    val exprAsts = if (ctx.expressionOrCommands() != null) {
```

Comment:
It would be more idiomatic to use `Option` instead of a null-comparison. There are quite a few examples in the code base. Please check look them up.

Reply:
Done